src/main/java/org/folio/service/transactions/batch/BatchPendingPaymentService.java

```diff
@@ -89,9 +89,11 @@ private void updateEncumbranceToCancelTransaction(Transaction encumbrance, doubl
   }
 
   private void updateEncumbranceToApplyTransaction(Transaction encumbrance, double amount, CurrencyUnit currency) {
+    if (encumbrance.getEncumbrance().getStatus() == Encumbrance.Status.UNRELEASED) {
+      encumbrance.setAmount(subtractMoney(encumbrance.getAmount(), amount, currency));
+    }
     encumbrance.getEncumbrance().setAmountAwaitingPayment(sumMoney(
```

Comment:
Is it fine if we have these updated for Released encumbrance transactions?

Reply:
Something tells me retuning early on Released status may be applicable here, what do you think?

Reply:
I think it makes sense, at least we will keep the same pattern as for updateEncumbranceToCancelTransaction()
Probably this if was missed initially for this method.

Reply:
Checked also our python script to fix finance issues.
We have command unrelease_open_orders_encumbrances_with_nonzero_amounts()
https://github.com/folio-org/mod-finance-storage/blob/master/scripts/fix_encumbrances/fix_encumbrances_quesnelia.py#L626

So it means that if encumbrance is released and has some non zero amount - it is not valid situation that is fixed by this script by unreleasing this encumbrance.